src/hoc/withTheme.js

```diff
@@ -93,6 +113,8 @@ const wrapWithTheme = (Component: ReactClass<any>) => {
     }
   }
 
+  polyfill(WithTheme)
```

Comment:
@mxstbr out of scope for this PR, but do you think we should ship a bundle in the future that drops these polyfills for decreased bundlesizes if someone really needs that? could be quite trivial to achieve using rollup

Reply:
My assumption was that for React v16.3+ the polyfill is turned into a noop, and you can't run React 16.2- without it, so who would ever use that extra bundle? Not sure if that's the case

Reply:
@mxstbr the concern was more around whether we'd want to reduce the bundlesize if possible, but I'm fine with always including it personally

Reply:
- User on React v16.3+: Polyfill is a noop and gets optimized out, no bundle size increase
- User on React v16.2-: Polyfill works to mask the new API, if you remove the polyfill from the bundle your app stops working

Maybe I'm misunderstanding what you're saying, there is no way to use that bundle, right?

Reply:
Ah really? If it's being optimised out that'd be perfect. Sorry for raising this then :)

Reply:
I don't think it could get optimized out—how would that work? Tools like webpack can't do static analysis like this, you'd need something like Prepack.

Reply:
That said it's likely that many libs will use the polyfill so as long as the version is in the same range, the bundle size cost will get amortized between them.